packages/blueprints/test-blueprint/src/blueprint.ts

```diff
@@ -14,6 +20,65 @@ import defaults from './defaults.json';
  * 5. The 'Options' member values defined in 'defaults.json' will be used to populate the wizard selection panel with default values
  */
 export interface Options extends ParentOptions {
+  /**
+   * @collapsed
+   */
+  multiSelectors?: {
+    stringMulti: MultiSelect<'First option' | 'A' | 'B' | 'B' | 'C'>;
+    numberMulti: MultiSelect<1 | 10 | 100 | 9999>;
+    /**
+     * This allows you to select one or more of any of the existing source repos in the project
+     */
+    sourceMulti?: MultiSelect<SourceRepository>;
+
+    /**
+     * This allows you to select one or more of any of the existing BlueprintInstantiations in the project
+     */
+    instantiationMulti?: MultiSelect<BlueprintInstantiation>;
+  };
+
+  /**
+   * These are touple parings.
```

Comment:
NIT: change touple to tuple in all occurences.

Reply:
ha! I need to learn to spell